app/helpers/policies_helper.rb

```diff
@@ -254,4 +254,31 @@ def shortened_randomised_people_voting_on_policy(policy, max_people:)
     # return the chosen people and the number of people not included
     [chosen_people, people.length - chosen_people.length]
   end
+
+  def people_under_policy_category(policy, category)
+    distances = policy.policy_person_distances.currently_in_parliament
+
+    # Insert people into each category
+    rep = []
+    distances.each do |ppd|
+      rep << ppd.person if ppd.category.to_s == category
+    end
+    rep
+  end
+
+  def card_title_from_category(policy, category)
+    case category.to_sym
+    when :not_enough
+      "We can't say anything concrete about how they voted on #{policy.name}"
+    else
+      "See the representatives who #{category_words_sentence(category.to_sym)}"
```

Comment:
"representatives" only refers to member of the lower house. Is there any reason this couldn't just say "See who #{category_words_sentence(category.to_sym)}"?